packages/cli/package.json

```diff
@@ -46,6 +46,9 @@
   "publishConfig": {
     "access": "public"
   },
+  "imports": {
+    "#net": "./src/net/index.mts"
```

Comment:
Kind of wild that you can import from a .mts file

Reply:
its a bit broken honestly. @pashakondratyev has tickets. we have to be careful with this mechanism.